src/testing/mock.rs

```diff
@@ -520,14 +521,15 @@ pub(crate) type HonestMember<'a> = Member<'a, Hasher64, Data, DataIO, KeyBox, Sp
 pub(crate) fn configure_network(
     n_members: usize,
     reliability: f64,
-) -> (UnreliableRouter, Vec<Option<Network>>) {
+    connected_peers: impl IntoIterator<Item = NodeIndex>,
```

Comment:
Consider making a separate function `configure_network_with_connected_peers` or something like that, because this argument is very boilerplatey for almost all uses of this function. Of course `configure_network` would call the other function with the `1..n_members.map(NodeIndex)` argument.

Reply:
Removed. Droping instances of network was enough to achive similar result.